src/asynch.rs

```diff
@@ -0,0 +1,94 @@
+//! Use APA102 leds via SPI with asynchronous writing of data via the
+//! [`embedded_hal_async::spi::SpiBus`](https://docs.rs/embedded-hal-async/latest/embedded_hal_async/spi/trait.SpiBus.html) trait.
+//!
+//! - For usage with `smart-leds`
+//! - Implements the `SmartLedsWriteAsync` trait
+//!
+//! Doesn't use the native brightness settings of the apa102 leds, since that
+//! runs at a much lower pwm frequency and thus nerfes the very high color pwm
+//! frequency. (According to Adafruit)
+
+use crate::PixelOrder;
+
+use embedded_hal_async::spi::SpiBus;
+use smart_leds_trait::{RGB8, SmartLedsWriteAsync};
+
+pub struct Apa102<SPI> {
```

Comment:
Could you rename the async `Apa102` struct to something akin to `Apa102Async` to avoid confusion?

Reply:
Yes good idea

Reply:
done